test/test.erl

```diff
@@ -28,19 +28,26 @@ gen_should_pass() ->
     }.
 
 gen_should_fail() ->
-    map_erl_files(
-      fun(File) ->
-              fun() ->
-                      Errors = gradualizer:type_check_file(File, [return_errors]),
-                      %% Test that error formatting doesn't crash
-                      Opts = [{fmt_location, brief},
-                              {fmt_expr_fun, fun erl_prettypr:format/1}],
-                      lists:foreach(fun({_, Error}) -> typechecker:handle_type_error(Error, Opts) end, Errors),
-                      {ok, Forms} = gradualizer_file_utils:get_forms_from_erl(File, []),
-                      ExpectedErrors = typechecker:number_of_exported_functions(Forms),
-                      ?assertEqual(ExpectedErrors, length(Errors))
-              end
-      end, "test/should_fail").
+    {setup,
+        fun() ->
+            %% user_types.erl is referenced by remote_types.erl.
```

Comment:
by opaque.erl

Or, rewrite this and the other instance as
user_types is referenced by other modules under test.

Reply:
👍